internal/cli/kraft/cloud/instance/restart/restart.go

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2025, Unikraft GmbH and The KraftKit Authors.
+// Licensed under the BSD-3-Clause License (the "License").
+// You may not use this file except in compliance with the License.
+
+package restart
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	kraftcloud "sdk.kraft.cloud"
+
+	"kraftkit.sh/cmdfactory"
+	"kraftkit.sh/config"
+	"kraftkit.sh/internal/cli/kraft/cloud/instance/start"
+	"kraftkit.sh/internal/cli/kraft/cloud/instance/stop"
+	"kraftkit.sh/internal/cli/kraft/cloud/utils"
+)
+
+type RestartOptions struct {
+	All    bool                  `long:"all" short:"a" usage:"restart all instances"`
+	Auth   *config.AuthConfig    `noattribute:"true"`
+	Client kraftcloud.KraftCloud `noattribute:"true"`
+	Force  bool                  `long:"force" short:"f" usage:"Force stop the instance(s)"`
+	Metro  string                `noattribute:"true"`
+	Token  string                `noattribute:"true"`
+	Wait   time.Duration         `local:"true" long:"wait" short:"w" usage:"Timeout to wait for the instance to stop and/or start (ms/s/m/h)"`
+}
+
+func NewCmd() *cobra.Command {
+	cmd, err := cmdfactory.New(&RestartOptions{}, cobra.Command{
+		Short:   "Restart instance(s)",
+		Use:     "restart [FLAGS] [UUID|NAME [UUID|NAME]...]",
+		Args:    cobra.ArbitraryArgs,
+		Aliases: []string{"str"},
```

Comment:
Is this correct, would have used rr as an alias

Reply:
whoops yeah copy pasta mistake from `start`